pkg/store/hintspb/hints.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+syntax = "proto3";
+package hintspb;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "hintspb";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_getters_all) = false;
+
+// Do not generate XXX fields to reduce memory.
+option (gogoproto.goproto_unkeyed_all) = false;
+option (gogoproto.goproto_unrecognized_all) = false;
+option (gogoproto.goproto_sizecache_all) = false;
+
+message SeriesResponseHints {
+    // queried_blocks is the list of blocks that have been queried.
```

Comment:
gRPC comments for fields and messages should start with 3 dashes. I learned that at Improbable... but at this point I am not sure why though, would need to dive deeper ;p @mwitkow might know ^^

Let's change for consistency now?